Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/SyntaxReceiver.cs

```diff
@@ -61,12 +67,26 @@ public void OnVisitSyntaxNode(GeneratorSyntaxContext context)
             if (context.Node is ClassDeclarationSyntax classDeclarationSyntax && classDeclarationSyntax.AttributeLists.Count > 0)
             {
                 // Get the symbol being declared by the class, and keep it if its annotated
-                var methodSymbol = context.SemanticModel.GetDeclaredSymbol(classDeclarationSyntax);
+                var methodSymbol = ModelExtensions.GetDeclaredSymbol(
+                    context.SemanticModel,
+                    classDeclarationSyntax);
                 if (methodSymbol.GetAttributes().Any(attr => attr.AttributeClass.Name == nameof(LambdaStartupAttribute)))
                 {
                     StartupClasses.Add(classDeclarationSyntax);
                 }
             }
+            
+            if (context.Node is MethodDeclarationSyntax methodDeclaration)
+            {
+                var model = ModelExtensions.GetDeclaredSymbol(
```

Comment:
Similar comment as above. Just call the method from `context.SemanticModel.GetDeclaredSymbol`.